Testing/functional/tests/cypress/integration/e2e/timeline/mobile/laboratoryOrder.js

```diff
@@ -0,0 +1,64 @@
+const { AuthMethod } = require("../../../../support/constants");
+
+function selectShouldBeVisible(selector) {
```

Comment:
Suggest renaming to `selectorShouldBeVisible` - also applies to the UI test file.